scripts/pre_commit_linter.py

```diff
@@ -21,6 +21,9 @@
 it will lint all JavaScript files in that directory; otherwise,
 it will only lint files that have been touched in this commit.
 
+This script ignores all filepaths contained within the excludeFiles
```

Comment:
Can you also amend the description above to explain that it lints both JavaScript and Python files? (You can omit the JSCS part; it's not that relevant.)